### DISCUSSION TITLE
Connecting to robot UR_10

### DISCUSSION THREAD

**Discussion by Nikhil-Singhal-06** [visose/Robots#200] [Q&A]
Hi all :)

I am trying to connect to my university robot UR_10 with Grasshopper Remote connection, but I get the following error message.

![Error](https://user-images.githubusercontent.com/87761277/202710287-91acc95c-5c42-4d26-9bec-74810e46a8af.png)

Has anyone had the same experience or know the [solution?

Thanks :)

**Comment by visose** [comment]
Double check that the robot IP is 192.168.3.12 and that the IP in your computer is set manually to a range in the 192.168.3.*, such as 192.168.3.10.
Then open a terminal and type "ping 192.1683.12" to see if you can reach the robot.

**Comment by rizvi-riaz** [comment]
Hi all,
I’m using Rhino 8 to connect remote to UR10e robot. The connection seems good (confirmed via ping), but after uploading commands several times, the robot doesn’t execute . Occasionally, saving as a new file temporarily resolves the issue, but it stops again. Interestingly, it works perfectly on some laptops but not on others.

We’ve spent a lot of time troubleshooting this without success. Would be great if you can provide any suggestions or insights into what might be causing this issue?

Thanks in advance!

**Reply by visose** [reply]
From what you describe I can't think of any suggestion. If you can, try to find out more information on when it works and when it doesn't and provide more detail. 
I also don't think that saving a program using the Save component would have any effect on the Remote component.

**Reply by rizvi-riaz** [reply]
Thanks, I will keep you posted:)

**Reply by LoyWeiWin** [reply]
Perhaps there are too many target points? I believe you can send approximately 2000 target points at most via remote connection; however, exceeding this limit may prevent the file from being transmitted.

**Comment by visose** [comment]
Closing this as stale. The original remote-connection question and later follow-up have not produced a current reproducible case. If this still fails on a current Robots/Rhino/UR controller version, please open a new discussion for support, or file an Issue if you can provide a minimal reproducible bug with exact versions, network setup, and the generated program.
